### ISSUE TITLE
Baron Longshore doesn't spawn randomly.

### ISSUE THREAD

**Issue by placoto99** [LightsHope/issues#470]
https://classicdb.ch/?npc=3467
Baron Longshore doesn't spawn randomly like it used to do in officials.

**Comment by volvodude** [comment]
he always respawns in the same spot

**Comment by placoto99** [comment]
Yeah that's the issue dude. He is supposed to spawn randomly. Near each camps near tents.

**Comment by placoto99** [comment]
proof of the second location there is 3 or 4 as I said https://www.youtube.com/watch?v=iab9hbGLSvk

**Comment by placoto99** [comment]
This is more vanilla-like ... https://vanilla-twinhead.twinstar.cz/?npc=3467

**Comment by volvodude** [comment]
Hi placoto99

Im just saying Ive been playing on lightbringer and have killed guy 15 - 20
times. My experience is he is always in the same spot. That's all

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#m_8286228210839155763_DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by placoto99** [comment]
Omfg that's why it's a bug wtf... It shouldn't spawn at the same location like it does actually...
